### DISCUSSION TITLE
Rewrite Dynamic URL to Static URL

### DISCUSSION THREAD

**Discussion by amender-technologies** [searx/searx#3468] [Installation issue: custom]
Hello Team,
We have one instance installed at [https://web.zoopzoopgo.com](https://web.zoopzoopgo.com).
We are searching for a solution to Rewrite Dynamic URL to Static URL of search results.
I have personally search all the possible places on internet but could not find anything.
Can you please help on this? Thanks

**CURRENT SEARCH RESULT URL:** https://web.zoopzoopgo.com/search?q=SEARCH+TERM&advanced_search=on&category_general=on&time_range=&language=en-IN

**EXPECTED SEARCH RESULT URL:** https://web.zoopzoopgo.com/search/q/SEARCH-TERM/advanced_search/on/category_general/on/language/en-IN

**Comment by unixfox** [comment]
Should be doable with some NGINX rules. Not that easy to do though.
